### PR TITLE
UBER-1160: Open vacancy panel when it’s opened from applicant

### PR DESCRIPTION
**Pull request by d-uzlov**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does a local formatting is applied (rush format)
* [x] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [x] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [x] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

File: plugins/recruit-resources/src/components/VacancyCard.svelte [hunk 1]
```diff
@@ -73,7 +81,7 @@
   {/if}
   {#if vacancy}
     <!-- svelte-ignore a11y-click-events-have-key-events -->
-    <NavLink {disabled} space={vacancy._id} app={recruitId}>
+    <NavLink {disabled} space={vacancyId} app={recruitId}>
```
Comment (BykhovDenis) [inline plugins/recruit-resources/src/components/VacancyCard.svelte]: May be better use DocNavLink ?)

Comment (ThetaDR) [reply plugins/recruit-resources/src/components/VacancyCard.svelte]: I agree :D

**Review by BykhovDenis — APPROVED** [review]

**Review by ThetaDR — APPROVED** [review]

**Comment by ThetaDR** [comment]
@d-uzlov update the branch and check uitests please

**Review by ThetaDR — APPROVED** [review]

**Comment by haiodo** [comment]
p1

**Comment by haiodo** [comment]
p2

**Pull request merged**